src/PaymentSuite/RedsysBundle/Controller/PaymentController.php

```diff
@@ -72,7 +72,7 @@ public function executeAction()
         return new Response(
             $this
                 ->templatingEngine
-                ->render('RedsysBundle:Redsys:process.html.twig', [
+                ->render('GestpayBundle:Redsys:process.html.twig', [
```

Comment:
I think the refactor command does something wrong here :-P

Reply:
Hmmm yeah it seems so, I was not aware of this. I'll change it and pass the whole test suite